src/server/scripts/Outland/BlackTemple/illidari_council.cpp

```diff
@@ -337,11 +323,13 @@ class boss_gathios_the_shatterer : public CreatureScript
                     events.ScheduleEvent(EVENT_SPELL_AURA, 30000);
                     break;
                 case EVENT_SPELL_HAMMER_OF_JUSTICE:
-                    if (Unit* target = SelectTarget(SelectTargetMethod::Random, 0, HammerOfJusticeSelector(me)))
-                    {
-                        me->CastSpell(target, SPELL_HAMMER_OF_JUSTICE, false);
-                        events.ScheduleEvent(EVENT_SPELL_HAMMER_OF_JUSTICE, 20000);
-                        break;
+                    if (Unit* target = me->GetVictim()) {
+                        if (target->GetTypeId() == TYPEID_PLAYER && me->IsInRange(target, 10.0f, 40.0f, true))
+                        {
+                            me->CastSpell(target, SPELL_HAMMER_OF_JUSTICE, false);
+                            events.ScheduleEvent(EVENT_SPELL_HAMMER_OF_JUSTICE, 20000);
```

Comment:
use 20s instead

Reply:
ok thanks:)
PS: (I used milliseconds because the other ScheduleEvents methods in the file use them)

Reply:
yeah I understand. but we want to move to using chrono

Reply:
As it is easier to read